### DISCUSSION TITLE
Adjusted Bounce Rate

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#193] [Anything Else]
"It would be really nice to be able to say someone who stays on the page more than 10 seconds is not a bounce"

**Comment by tcurdt** [comment]
So much this. I am sending custom events after a certain amount of time - but it does not seem to have any effect on the bounce rate. That is a problem.

**Reply by metmarkosaric** [reply]
Thanks @tcurdt! We'd prefer to move away from bounce rate completely and do something much more useful. What do you think about #591 ?

**Reply by tcurdt** [reply]
Would be perfect! That's basically what I do manually with a custom event anyway.

**Reply by metmarkosaric** [reply]
Nice to hear that! Next time we focus the development on adding new metrics we will probably place this change as high priority alongside time on page, entry/exit pages and region/state/city. Right now the development focus is on building the API! Stay tuned!

**Comment by Dan0sz** [comment]
Hi @metmarkosaric,

Sorry for taking so long to post my solution here.

So, basically what I'm doing is what @tcurdt mentioned earlier as well. I don't know what his contents of the event are, but this is what my tracking code looks like:

````
<script defer data-domain="domain.name" data-api="https://url.to/proxy" src="https://link.to/file.js"></script>
<script>
    window.plausible = window.plausible || function() {
        (window.plausible.q = window.plausible.q || []).push(arguments)
    }
    setTimeout("plausible('Adjusted Bounce Rate', { props: { duration: '40' } });", 40000);
</script>
````

Maybe the `props` and `duration` attributes are making the difference? With me it definitely works. I've implemented this method into [my WordPress plugin](https://wordpress.org/plugins/host-analyticsjs-local) and I know it's working with others as well.

A few things to mention about my setup: I use a proxy to bypass ad blockers and I've saved the file to my server. I doubt this'll make a difference, but I just thought I'd mention it.

**Reply by metmarkosaric** [reply]
thanks for sharing @Dan0sz!

**Reply by Dan0sz** [reply]
Btw. I forgot to mention: I did add [a custom event goal](https://plausible.io/docs/custom-event-goals) to my Plausible Dashboard, called Adjusted Bounce Rate.
